config.json

```diff
@@ -0,0 +1 @@
+{"address":"546741f9deaabaa141912c9fae316db855b897cf","id":"f156d54c-7ae9-4f39-8930-c10460233763","version":3,"Crypto":{"cipher":"aes-128-ctr","cipherparams":{"iv":"f45d4495a917452bb0730430d7d477ce"},"ciphertext":"c239c113425e4dd8c54850703c60c18104377a9dc253676ecd5dee9dfafe1469","kdf":"scrypt","kdfparams":{"salt":"8102ce9d6060096de51d6cb4f218b2a33f309360db0dc1b7ef1b41a6c7d5b736","n":131072,"dklen":32,"p":1,"r":8},"mac":"0f4bce48637692ef09080fbb4fe083c47cce800a2817759bb2a75a23ceeb3f92"},"x-ethers":{"client":"ethers.js","gethFilename":"UTC--2021-02-01T02-34-09.0Z--546741f9deaabaa141912c9fae316db855b897cf","mnemonicCounter":"e6ca667e1314c7b842abc24138a1b8fd","mnemonicCiphertext":"31f95a06672054dc0a02cb6e3dace5f2","path":"m/44'/60'/0'/0/0","locale":"en","version":"0.1"}}
```

Comment:
what's this file for ?

Reply:
should be accidentally checked in to one of the commit, will remove it.